compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/lower/AbstractSuspendFunctionsLowering.kt

```diff
@@ -80,9 +80,10 @@ abstract class AbstractSuspendFunctionsLowering<C : CommonBackendContext>(val co
     // Suppress since it is used in native
     @Suppress("MemberVisibilityCanBePrivate")
     protected fun IrCall.isReturnIfSuspendedCall() =
-        symbol.owner.run { fqNameWhenAvailable == context.internalPackageFqn.child(Name.identifier("returnIfSuspended")) }
+        symbol.isPublicApi && context.returnIfSuspendedSignature.equalsWithNames(symbol.signature)
```

Comment:
Sorry, I didn't notice the first time that there's already `returnIfSuspended` in `Symbols`. Maybe do something like this instead then: `context.ir.symbols.returnIfSuspended.signature == symbol.signature`?